lib/galaxy/schema/schema.py

```diff
@@ -3805,3 +3807,23 @@ class PageSummaryList(RootModel):
 class MessageExceptionModel(BaseModel):
     err_msg: str
     err_code: int
+
+
+class SanitizedString(str):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, value):
+        if isinstance(value, str):
+            return cls(sanitize_html(value))
```

Comment:
I don't think we should be doing any HTML sanitization on the incoming side.

Reply:
So should this validator be removed and use str as type?

Reply:
Let's keep it for now. I think we still have a whole bunch of legacy visualizations and I don't know if they can access things like the title.